pandas/tests/test_base.py

```diff
@@ -998,6 +998,42 @@ def test_validate_bool_args(self):
             with pytest.raises(ValueError):
                 self.int_series.drop_duplicates(inplace=value)
 
+    def test_get(self):
```

Comment:
this should be in pandas/tests/series/test_indexing (and pandas/tests/frame/test_indexing)

Reply:
Moved it to pandas/tests/series/indexing/test_indexing and parameterized the test

Note for dataframes, `df.get()` has a meaning to get the *column*, not the row, so I didn't put a test there.